projects/origin-release/pipeline.yaml

```diff
@@ -221,6 +221,36 @@ objects:
     triggers:
     - imageChange: {}
       type: ImageChange
+- apiVersion: v1
```

Comment:
Out of curiosity, what triggers this when a now golang patch version becomes available?

Reply:
Humans